telegram/ext/_application.py

```diff
@@ -466,7 +466,7 @@ async def start(self) -> None:
                 _logger.debug('Loop for updating persistence started')
 
             if self.job_queue:
-                await self.job_queue.start()
+                await self.job_queue.start()  # type: ignore[union-attr]
```

Comment:
can we get rid of the comment by changing the above line to `if self.job_queue is not None`? same below

Reply:
didn't fix it.. here's the error: `error: Item "object" of the upper bound "Optional[JobQueue]" of type variable "JQ" has no attribute "start"`